src/sync/transforms/pull-request.ts

```diff
@@ -0,0 +1,85 @@
+import parseSmartCommit from '../../transforms/smart-commit';
+import { GitHubAPI } from 'probot';
+
+// TODO: better typings in file
+function mapStatus({ state, merged_at }): string {
+  if (state === 'merged') return 'MERGED';
+  if (state === 'open') return 'OPEN';
+  if (state === 'closed' && merged_at) return 'MERGED';
+  if (state === 'closed' && !merged_at) return 'DECLINED';
+  return 'UNKNOWN';
```

Comment:
For some reason any pull request I create is falling to meet any of the conditional checks here and is being labelled as 'UNKNOWN'.

![Screen Shot 2021-06-08 at 10 07 08 am](https://user-images.githubusercontent.com/37155488/121103167-9baaf080-c842-11eb-8df1-2e3890fd6900.png)

Saw the above when I created a PR (should have been `OPEN`) and again when I merged it (`MERGED`).

Reply:
Yup same here 

<img width="269" alt="Screen Shot 2021-06-08 at 2 48 02 pm" src="https://user-images.githubusercontent.com/58801956/121124777-d4f65700-c868-11eb-917e-bcc1a4946960.png">

Reply:
Fixed.